src/api/spec/support/shared_examples/features/beta/user_tab.rb

```diff
@@ -41,6 +41,8 @@
     end
 
     scenario 'Add non existent user' do
+      skip_on_mobile
```

Comment:
Can't we avoid skipping the test somehow?

Reply:
As the description says, in [this comment](https://github.com/openSUSE/open-build-service/pull/9559#pullrequestreview-413770036) it was considered, to scroll down until an element that should be clicked appeared into the visual area.

I'm open to other alternatives, too.

Reply:
doesn't has to happen here and now right?

Reply:
I found a solution for this, scrolling to the element will work. See the commit I pushed.